cloud_topic: /cloud_registered
odom_topic: /Odometry
map_frame: map
local_frame: odom
static_map_path: "" # Leave empty to disable auto-load
update_hz: 10.0

rough_scan_resolution: 0.25
rough_map_resolution: 0.25
rough_max_iteration: 5
rough_score_thresh: 0.2

refine_scan_resolution: 0.1
refine_map_resolution: 0.1
refine_max_iteration: 10
refine_score_thresh: 0.1